ws/ws.d.ts

```diff
@@ -8,6 +8,7 @@
 declare module "ws" {
     import * as events from 'events';
     import * as http from 'http';
+    import * as http from 'https';
```

Comment:
Funny.

LGTM

On Sep 14, 2016 06:02, "TonyYang" <notifications@github.com> wrote:

> In ws/ws.d.ts
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11213#discussion_r78679937>
> :
>
> > @@ -8,6 +8,7 @@
> >  declare module "ws" {
> >      import * as events from 'events';
> >      import * as http from 'http';
> > +    import * as http from 'https';
>
> Hi @mabels <https://github.com/mabels>,
> it should be import * as https from 'https';. Please improve it. 😄
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11213/files/4b724582216cfec426735562e9f7ad3bbd299a4e#r78679937>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA6HdBIm7PuCNJfCvHSHKF5Ndn9iwZW7ks5qp2OxgaJpZM4J8RfN>
> .
>